From: Klee Dienes <klee.dienes@hadronindustries.com>
Date: Wed, 4 Dec 2013 01:44:57 -0500
Subject: Allow specifying library type with TARGET_LIBRARYTYPE.

---
 CMakeModules/OsgMacroUtils.cmake | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/CMakeModules/OsgMacroUtils.cmake b/CMakeModules/OsgMacroUtils.cmake
index 85e23c5..ed7da03 100644
--- a/CMakeModules/OsgMacroUtils.cmake
+++ b/CMakeModules/OsgMacroUtils.cmake
@@ -257,6 +257,10 @@ MACRO(SETUP_PLUGIN PLUGIN_NAME)
 
     ## we have set up the target label and targetname by taking into account global prfix (osgdb_)
 
+    IF(NOT TARGET_LIBRARYTYPE)
+      SET(TARGET_LIBRARYTYPE MODULE)
+    ENDIF()
+
     IF(NOT TARGET_TARGETNAME)
             SET(TARGET_TARGETNAME "${TARGET_DEFAULT_PREFIX}${TARGET_NAME}")
     ENDIF(NOT TARGET_TARGETNAME)
@@ -277,7 +281,7 @@ MACRO(SETUP_PLUGIN PLUGIN_NAME)
     # here we use the command to generate the library
 
     IF   (DYNAMIC_OPENSCENEGRAPH)
-        ADD_LIBRARY(${TARGET_TARGETNAME} MODULE ${TARGET_SRC} ${TARGET_H})
+        ADD_LIBRARY(${TARGET_TARGETNAME} ${TARGET_LIBRARYTYPE} ${TARGET_SRC} ${TARGET_H})
     ELSE (DYNAMIC_OPENSCENEGRAPH)
         ADD_LIBRARY(${TARGET_TARGETNAME} STATIC ${TARGET_SRC} ${TARGET_H})
     ENDIF(DYNAMIC_OPENSCENEGRAPH)
