Author: <alberto@pantallon>
Description: No description.
Index: osggit/OpenSceneGraph/src/osgPlugins/CMakeLists.txt
===================================================================
--- osggit.orig/OpenSceneGraph/src/osgPlugins/CMakeLists.txt	2013-06-21 17:21:17.000000000 +0200
+++ osggit/OpenSceneGraph/src/osgPlugins/CMakeLists.txt	2013-06-21 17:21:17.000000000 +0200
@@ -237,9 +237,9 @@
     ADD_SUBDIRECTORY(qfont)
 ENDIF()
 
-IF(ZLIB_FOUND)
-    ADD_SUBDIRECTORY(zip)
-ENDIF()
+# IF(ZLIB_FOUND)
+#     ADD_SUBDIRECTORY(zip)
+# ENDIF()
 
 IF(RSVG_FOUND)
     ADD_SUBDIRECTORY(svg)
